osmomath/math.go

```diff
@@ -171,3 +172,12 @@ func PowApprox(base sdk.Dec, exp sdk.Dec, precision sdk.Dec) sdk.Dec {
 	}
 	return sum
 }
+
+// Max returns the maximum value between two Ordered types.
+func Max[T constraints.Ordered](x, y T) T {
```

Comment:
Please make sure that this is not a duplicate of a function that was moved

Reply:
We are currently using this func here https://github.com/osmosis-labs/osmosis/blob/5d72afb0cd14a3ba5916d0f084def0075a528e3b/x/concentrated-liquidity/tick.go#L49-L65